extension/fts/src/include/fts_extension.h

```diff
@@ -0,0 +1,81 @@
+#pragma once
+
+#include "extension/extension.h"
+
+namespace kuzu {
+namespace fts_extension {
+
+class FTSExtension final : public extension::Extension {
+public:
+    static constexpr char EXTENSION_NAME[] = "FTS";
+
+public:
+    static void load(main::ClientContext* context);
+
+    static constexpr const char* STOP_WORDS[] = {"a", "a's", "able", "about", "above", "according",
```

Comment:
Should we call this EN_STOP_WORDS? This will never be overwritten, right?

Reply:
renamed